extra/resources/SysInfo.in

```diff
@@ -90,7 +90,7 @@ Units:
 <content type="string" default="0s" />
 </parameter>
 
-<parameter name="disks">
+<parameter name="disks" required="1">
```

Comment:
disks is not actually required because the for loop in SysInfoStats() always checks at least "/" (which is a questionable design, that should maybe have been the default instead, but for backward compatibility we should leave that alone)